[BUGFIX/PATCH]: Markdown Documentation Cleanup (#2977)
FunkinCrew/Funkin: README.md

# Credits and Special Thanks

> The full credits can be found in-game, or wherever the credits.json file is.

Reviewer: `The full` can be reverted back to `Full`, and here's the link to `credits.json` as follows:
```suggestion
Full credits can be found in-game, or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).
```

## Programming
- [ninjamuffin99](https://twitter.com/ninja_muffin99) - Lead Programmer
- [EliteMasterEric](https://twitter.com/EliteMasterEric) - Programmer
- [MtH](https://twitter.com/emmnyaa) - Charting and Additional Programming
- [Jenny Crowe](https://twitter.com/JennyCrowe) - Additional Programming
- [hazel](https://twitter.com/hazel) - Additional Programming, Devops and Additional Internal Tooling
- [Mike Welsh](https://twitter.com/MikeWelsh) - Additional Programming
- [Saharan](https://twitter.com/Saharan) - Additional Programming
- [Ian Harrigan](https://twitter.com/IanHarrigan) - Additional Programming

> - [Osaka Red LLC: Thomas J Webb](https://twitter.com/ThomasJWebb) - Additional Programming

Reviewer: I believe that the aforementioned people here have been attached to incorrect Twitter accounts or accounts that simply don't exist. 

Maybe someone from the team should chime in and examine and see if they can correct the mistakes and say who does or doesn't have Twitter and/or any other social media platform.

- [GeoKureli](https://twitter.com/Geokureli/) - Additional Programming

> - [Will Blanton](https://twitter.com/WillBlanton) - Additional Programming

Reviewer: Abovementioned comment also applies to this user.

- [Cheemsandfriends](https://twitter.com/Cheemsandfriends) - FlxAnimate
- Our contributors on GitHub

## Art / Animation / UI
- [PhantomArcade3K](https://twitter.com/phantomarcade3k) - Artist and Animator
- [Evilsk8r](https://twitter.com/evilsk8r) - Art
- [beck](https://twitter.com/beck) - Art, Cutscene Cleanup, Cutscene Background Art
- [Red Minus](https://twitter.com/RedMinus) - Additional Background Design
- [Figburn](https://twitter.com/Figburn) - Cutscene Animation
- [Sade](https://twitter.com/Sade) - Cutscene Animation
- [Topium](https://twitter.com/Topium) - Cutscene Animation
- [BlairTheUnseriousGuy](https://twitter.com/BlairTheUnseriousGuy) - Cutscene Animation

> - [PennilessRagamuffin](https://twitter.com/PennilessRagamuffin) - Cutscene Animation

Reviewer: Same goes here, accounts are either incorrect or just straight up don't exist.

- [Moawling](https://twitter.com/moawko) - Week 6 Pixel Art
- [IvanAlmighty](https://twitter.com/IvanA1mighty) - Misc UI Design, Voice of Darnell

## Music
- [Kawaisprite](https://twitter.com/kawaisprite) - Lead Musician
- [Saruky](https://twitter.com/Saruky__) - Musician

> - [SixImpala](https://twitter.com/SixImpala) - Musician

Reviewer: Handle should be changed to:
```suggestion
- [SixImpala](https://twitter.com/siximpalasix) - Musician
```

- [Saster](https://twitter,com/sub0ru) - Featured Guest Musician for "Thorns Erect"
- [nuphory](https://twitter.com/nuphory) - Featured Guest Musician for "Results (Excellent)

> - [MtH](https://twitter.com/emmnyaa) - Featured Guest Musician for "Breakfast (Week 6 Version)

Reviewer 2: Make sure to add end quotes after Results and Breakfast!

Reviewer 2: Remember to add quotes to Results (Excellent) too :)

# Credits and Special Thanks

> Full credits can be found in-game or in the [credits.json](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json) file or in the [CREDITS.md](/docs/CREDITS.md) file.

Reviewer: ```suggestion
Simplified credits can be found in [CREDITS.md](/docs/CREDITS.md), while full credits can be found in-game or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).
```

Author: Yes thank you!

> Full credits can be found in-game, [CREDITS.md,](https://github.com/FunkinCrew/Funkin) or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).

Reviewer: ```suggestion
Full credits can be found in-game or in [CREDITS.md](https://github.com/FunkinCrew/Funkin).
```

> Full credits can be found in-game, [CREDITS.md,](https://github.com/FunkinCrew/Funkin) or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).

Reviewer 2: ```suggestion
Full credits can be found in-game or in the [CREDITS.md](/docs/CREDITS.md) file.
```
